src/lib/forms/Switch.svelte

```diff
@@ -50,6 +50,27 @@
 </div>
 
 <style>
+  @keyframes switch {
+    0% {
+      left: 0.5rem;
+      width: 1.5rem;
+    }
+    20% {
+      left: 0.75rem;
+      width: 2.5rem;
+      height: 1.25rem;
+    }
+    80% {
+      width: 1.5rem;
+      height: 1.52rem;
+      left: 1.5rem;
+    }
+    100% {
+      left: 1.5rem;
+      width: 1.5rem;
+    }
+  }
+
```

Comment:
Can you share an example video of what you based this on?

Reply:
realistically just eyeballing it. there isn't any specific documentation on values and the web components are not accurate to the current state of material you